### PR TITLE
copy button functionality resolved

### PR DESCRIPTION
**Pull request by balaji2210**

### CODE REVIEW

**Comment by arjungarg07** [comment]
Hey @balaji2210, we don't want a whole jquery cdn file to be downloaded for just a copy-paste functionality. Try implementing this using the window browser method. Also, it would be good if you provide a gif of you testing the functionality and add it in the description. Thanks

**Comment by balaji2210** [comment]
captured

**Comment by arjungarg07** [comment]
Well done.Thanks!

File: public/js/index.js [hunk 1]
```diff
@@ -62,9 +63,6 @@ function submitForm(e) {
 }
 
 function ClipBoard(result) {
-	navigator.clipboard.writeText(result);
-	$('#copied').slideToggle(200);
-	setTimeout(function () {
-		$('#copied').slideToggle(200);
-	}, 1000);
+	 navigator.clipboard.writeText(result);
+	document.getElementById('copy').innerText='Copied'
```
Comment (arjungarg07) [inline public/js/index.js:67]: Remove this change and make a separate PR for it. You shall also change the CSS to a lil bit darker respective to the current color of the btn.

**Review by arjungarg07 — CHANGES_REQUESTED** [review]

**Comment by arjungarg07** [comment]
Are you working on it? @balaji2210

**Comment by balaji2210** [comment]
> Are you working on it? @balaji2210

yes  I have made a new PR

**Pull request closed without merging**